src/pcapi/core/offers/api.py

```diff
@@ -191,3 +192,13 @@ def delete_stock(stock: Stock) -> None:
 
     if feature_queries.is_active(FeatureToggle.SYNCHRONIZE_ALGOLIA):
         redis.add_offer_id(client=app.redis_client, offer_id=stock.offerId)
+
+
+def update_mediation(mediation: Mediation, is_active: bool) -> Mediation:
+    mediation.populate_from_dict({"isActive": is_active})
```

Comment:
Je pense qu'on pourrait faire sans le `populate_from_dict()`, ici, qui n'apporte pas grand chose : 

```suggestion
    mediation.isActive = is_active
```